data/promotional-collections/highlights-japan.json

```diff
@@ -0,0 +1,227 @@
+{
+    "id": "highlights-japan",
+    "title": "Highlights from Japan",
+    "impactStatement": "A selection of research from scientists based in institutions in Japan.",
+    "summary": [
+        {
+            "type": "paragraph",
+            "text": "eLife has published papers on many tropical diseases, including malaria, Ebola, leishmaniases, Dengue and African sleeping sickness. The articles below have been selected by eLife editors to give a flavour of the breadth of research on tropical diseases published by the journal."
+        }
+    ],
+    "published": "2020-01-20T11:19:26Z",
+    "image": {
+        "banner": {
+            "alt": "",
+            "uri": "https://iiif.elifesciences.org/journal-cms/collection%2F2015-09%2F2d2e35c4-tropical-disease-elife-collections.png",
```

Comment:
no problem with the images being real URLs as it probably matches the rest of this project